### ISSUE TITLE
Lando 3.4.2 Crashing on Apple Macbook pro - with Apple M1 Chip

### ISSUE THREAD

**Issue by AbhayPai** [lando/lando#3228]
![Screen Shot 2021-12-02 at 9 17 39 PM](https://user-images.githubusercontent.com/14177170/144429450-5e6bdba4-874b-41cf-8cff-2c3c15479041.png)
![Screen Shot 2021-12-02 at 9 18 05 PM](https://user-images.githubusercontent.com/14177170/144429533-75980ff7-a149-4a1e-9529-f29e237c8452.png)

After Explicitly typing `lando composer install` it runs the command and exits. Not sure if this release issue, OS issue, or Composer issue. No Error logs are thrown. 

Not sure anyone else is facing the same or not in lando community.

**Comment by reynoldsalec** [comment]
@AbhayPai could you try running the relevant command with the `-vvv` option on (for example, `lando build -vvv`)? Would like to see if this is truly related to #3200 or separate issue.

**Comment by AbhayPai** [comment]
Thanks @reynoldsalec for responding and referring for more debugging steps here.

Providing screenshot of following commands:
1. `lando composer install -vvv`
![Screen Shot 2021-12-03 at 6 41 06 AM](https://user-images.githubusercontent.com/14177170/144514676-a3835319-f18f-44b6-940b-1b9367c9a0ce.png)
2. Application rebuild was executed without verbose and command used was `lando rebuild`. To confirm no other steps of execution was failed here expect composer giving error as segmentation fault.
![Screen Shot 2021-12-03 at 6 45 34 AM](https://user-images.githubusercontent.com/14177170/144515161-67446633-02b4-463f-8a77-5b64484e46f9.png)
3. Application rebuild executed with verbose and command used was `lando rebuild -vvv`

![Screen Shot 2021-12-03 at 6 48 52 AM](https://user-images.githubusercontent.com/14177170/144515474-4c843944-0ad7-407f-b794-44e617f564aa.png)
![Screen Shot 2021-12-03 at 6 48 10 AM](https://user-images.githubusercontent.com/14177170/144515401-13405c22-d62f-4c31-a9c5-b1fbaa79af44.png)

May i also know if there is any workaround for this, to run app ? Currently am using following lando configuration.

```sh
name: someappname
recipe: drupal8
config:
  via: apache
  php: '7.4'
  composer_version: 2.1.3
  drush: 9.7.1
  webroot: web/
 ```

**Comment by reynoldsalec** [comment]
@AbhayPai can you tell if it's failing on installing a specific package?

**Comment by AbhayPai** [comment]
Issue is happening while installing drupal core packages. To me it seems like similar to -- segfaults from curl -- is also causing trouble when using acli.

"Maybe helpful to reference, this same acli segfault issue surfaced in DDEV and was fixed here: [drud/ddev#3116](https://github.com/drud/ddev/issues/3116) (also by using the debian bullseye image)." - @sarahg

**Comment by sarahg** [comment]
I'm not sure if the `lando composer` issues are the same problem as the one with curl (#3200), but it seems possible. I don't know enough about how Composer works to say for sure.

But, our team has also ran into similar errors as @AbhayPai describes in this issue with `lando composer` on M1 Macs. We've found this one easier to workaround since a non-Lando "composer install" works OK. I'll ask around to see if we see this failing on any particular packages, but we have seen it on multiple Lando + WordPress sites we run.

**Comment by reynoldsalec** [comment]
@labboy0276 this is the other issue related to updating the PHP base images.

**Comment by reynoldsalec** [comment]
This should be solved with the Lando 3.6.0 release (currently marked as pre-release): https://github.com/lando/lando/releases/tag/v3.6.0

**Comment by AbhayPai** [comment]
Thanks a lot @reynoldsalec and @labboy0276 for looking into this. It works smoothly and working as expected. Cheers.
